### ISSUE TITLE
visual encoder issue

### ISSUE THREAD

**Issue by japharl** [martme/webaudio-modem#4]
When pressing down arrow, the encoding type will change instead of staying with the current type, leading to a sudden change in sound type.

**Comment by martme** [comment]
This is due to the accessibility defaults in the browser. When the dropdown has focus, the arrow keys can be used to change the value. However, if this also happens when the input does not have focus, it's possibly a bug!
